Sheepish-Squirrels/bangazon-api-sprint1
# created json files and database of customer and product data with faker

Fixes #13 

Changes proposed in this pull request:
- npm installs:
faker
dotenv
sqlite3
express
- created customer and product data with faker and stored to json files
- created function that renders sqlite table for products

Steps to test:
- check to make sure there is data stored in customers.json and products.json
- open bangazon.sqlite in DB Browser and confirm product data is stored

@SladeUtley
